test/e2e/storage/testsuites/volume_io.go

```diff
@@ -77,6 +77,10 @@ func (t *volumeIOTestSuite) getTestSuiteInfo() TestSuiteInfo {
 	return t.tsInfo
 }
 
+func (t *volumeIOTestSuite) skipUnsupportedTests(pattern testpatterns.TestPattern, driver TestDriver) {
+	skipVolTypePatterns(pattern, driver)
```

Comment:
feels like a basic test, whats the difference between this and the normal ones that mount/write

Reply:
most tests usually just read/write a few characters into a file.  These tests issue more IO, up to 1G of data.  Regarding IO to disk, do you see a reason why it would be different if you used inline vs preprovisioned vs dynamic pv?

Reply:
nope, looks good to skip